war/src/main/less/pluginSetupWizard.less

```diff
@@ -1,7 +1,6 @@
-@import (css) 'https://maxcdn.bootstrapcdn.com/font-awesome/4.4.0/css/font-awesome.min.css';
-@import (css) '../css/icons/icomoon.css';
-@import url(https://fonts.googleapis.com/css?family=Roboto:400,300,500,900,700);
-
+@import (css) '../css/font-awesome/css/font-awesome.min.css';
+@import (css) '../css/icomoon/css/icomoon.css';
```

Comment:
I'm investigating if Jenkins is using this icon library /cc @tfennelly 

My proposal is to remove it if no used.